airflow/providers/cohere/operators/embedding.py

```diff
@@ -58,11 +72,17 @@ def __init__(
         self.input_text = input_text
         self.timeout = timeout
         self.max_retries = max_retries
+        self.request_options = request_options
 
     @cached_property
     def hook(self) -> CohereHook:
         """Return an instance of the CohereHook."""
-        return CohereHook(conn_id=self.conn_id, timeout=self.timeout, max_retries=self.max_retries)
+        return CohereHook(
+            conn_id=self.conn_id,
+            timeout=self.timeout,
+            max_retries=self.max_retries,
```

Comment:
Should we use `max_retries=self.max_retries` in the same PR we are deprecating it? :)